Source/JavaScriptCore/runtime/FunctionExecutable.cpp

```diff
@@ -129,15 +131,25 @@ DEFINE_VISIT_OUTPUT_CONSTRAINTS(FunctionExecutable);
 
 FunctionExecutable* FunctionExecutable::fromGlobalCode(
     const Identifier& name, JSGlobalObject* globalObject, const SourceCode& source, LexicallyScopedFeatures lexicallyScopedFeatures,
-    JSObject*& exception, int overrideLineNumber, std::optional<int> functionConstructorParametersEndPosition)
+    JSObject*& exception, int overrideLineNumber, std::optional<int> functionConstructorParametersEndPosition, FunctionConstructionMode functionConstructionMode)
 {
+    if (overrideLineNumber == -1) {
```

Comment:
Can we get a named constant for this?

Reply:
Yeah, maybe we can use this `constexpr int NO_OVERRIDE_LINE_NUMBER = -1;`.